### ISSUE TITLE
Androguard is not maintained right now

### ISSUE THREAD

**Issue by reox** [androguard/androguard#833]
I used androguard for several years and was able to develop a few things for it.
Unfortunately, my work has changed and I'm no longer using androguard nor have I enough time to develop new features or fix bugs.
Some other developers have provided a few PR's but as you might have seen, the issue tracker is full with bugs and feature requests, which I can no longer work on.
Thus please note, that **androguard is currently not actively maintained!**

If someone wants to step in and work on androguard, please feel free to post your interest here, send PRs (which I might merge if I sporadically look into the repo) or simply fork the project for now.

**Comment by subho007** [comment]
@reox you have been the most active maintainer of the project, sad to see you leaving.

**Comment by adesnos** [comment]
I'm back, I will take care of that now !!

Thx all guys for all support during these years !!!

**Comment by reox** [comment]
@adesnos nice to see you back!

**Comment by adesnos** [comment]
@reox thx for all the work since you take this project ! Best wish for your new work

**Comment by subho007** [comment]
Welcome back @adesnos 👍 🎉

**Comment by ivan11paths** [comment]
There were many issues that were closed. The reason was this change in the project?

**Comment by adesnos** [comment]
Yes. I started this project 10 years ago and left many years ago in order to work on different subjects related to Android Security :)

I know that many users is using it and I closed all issues knowingly  because I know all bottlenecks of the python code, and I don't want to spent my time on it. I will switch all parsing file format to rust, and everything that TAKE a LOT of memory, cpu usage, etc. But of course I will keep up a python binding for easy usage as it is also the idea.

Expect really a better user experience, easy way to integrate into all pipelines that use Androguard in free/commercial tool.
